src/txmempool.cpp

```diff
@@ -217,10 +217,11 @@ void CTxMemPool::UpdateAncestorsOf(bool add, txiter it, setEntries &setAncestors
         UpdateChild(piter, it, add);
     }
     const int64_t updateCount = (add ? 1 : -1);
-    const int64_t updateSize = updateCount * it->GetTxSize();
+    const int64_t updateSize = it->GetTxSize();
+    assert(updateSize > 0);
```

Comment:
Why are you adding this? `GetTxSize` returns `size_t` and asserting `!= 0` is new behavior. I think this could be dropped.

Reply:
> `GetTxSize` returns `size_t` and asserting `!= 0` is new behavior.

Fixed.

> I think this could be dropped.

Let's keep `assert` for safety ;)